### ISSUE TITLE
Option not to flash console when Finicky is triggered

### ISSUE THREAD

**Issue by pichfl** [johnste/finicky#123]
I have yet to dig into the source code to check if there is a good reason this has to happen, but unless I'm debugging, I would much prefer not to flash the console window and dock icon every time I click a link.

It should be possible to prevent this by making the container a pure status bar app, I'll just can't remember the proper api from the top of my head.

**Comment by johnste** [comment]
- The dock icon shouldn't be visible, but there have been issues with this before. #78 Are you seeing it all the time?
- Generally the console window should be closed and will not show up when Finicky routes urls

**Comment by pichfl** [comment]
I do see only a flicker of both the dock icon. The animation expanding the dock doesn't even complete before it starts to collapse again. Same goes for the window, it's there enough to quickly steal the focus of the current window before returning it and finally having the new browser window to be focussed. 

I can do a quick video, not sure if QuickTime is grabbing enough frames though.

**Comment by johnste** [comment]
Would appreciate a video! 

Of course, without being able to reproduce the problem will make it very hard to fix.

**Comment by pichfl** [comment]
I did write a bit of Swift in the past, I mostly created this issue and the other two for myself 😅 will try to provide a video.

My config:

```js
module.exports = {
  defaultBrowser: 'Safari',
  handlers: [
    {
      match: finicky.matchHostnames(['localhost']),
      browser: 'Edge',
    },
    {
      match: finicky.matchHostnames(['meet.google.com']),
      browser: 'Google Chrome',
    },
    {
      match: /zoom.us\/j\//,
      browser: 'us.zoom.xos',
    },
    {
      match: finicky.matchHostnames(['teams.microsoft.com']),
      browser: 'com.microsoft.teams',
      url({ url }) {
        return {
          ...url,
          protocol: 'msteams',
        };
      },
    },
  ],
};
```

[Video Reproduction.mov.zip](https://github.com/johnste/finicky/files/4820395/Bildschirmvideo.aufnehmen.2020-06-23.um.17.35.54.mov.zip)

Note: The flickering drop shadow of the Tot window & the Dock that animates for no obvious reason.

**Comment by pichfl** [comment]
Having run the app straight from Xcode and having disabled any launching window inside the app (at least as far as I can follow), I think this might not be Finicky flashing, but the respective browser? Will continue investigating, not sure how I can slow down the whole thing so much that I can see what is happening.

**Comment by johnste** [comment]
Might be worth checking out if disabling [this line](https://github.com/johnste/finicky/blob/master/Finicky/Finicky/AppDelegate.swift#L208) ends up showing Finicky in the dock. It's a workaround for the icon to show up in the dock before.

**Comment by pichfl** [comment]
Yup, removing that line makes the icon appear for a few split seconds longer, actually showing the Finicky icon appearing. I wonder if you could have a daemon / cli type app that doesn't create windows at all and have a separate UI.

**Comment by johnste** [comment]
Should've been fixed in #78

**Comment by pichfl** [comment]
It was! Sorry for not closing this earlier.
